changelog.json

```
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "1.6.3": [
+      "[Added] Provide option to discard when files would be overwritten by a checkout - #6755. Thanks @mathieudutour!",
+      "[Fixed] Context menu doesn't open when right clicking on the edges of files in Changes list - #6296. Thanks @JQuinnie!",
+      "[Fixed] Display question mark image when there is no commit selected in dark theme - #6915. Thanks @say25!",
+      "[Fixed] No left padding for :emoji:/@user/#issue autocomplete forms. - #6895. Thanks @murrelljenna!",
+      "[Fixed] Reinstate missing image and fix illustration to work in the dark theme when there are no local changes - #6894",
+      "[Fixed] Resizing the diff area preserves text selection range - #2677",
+      "[Fixed] Text selection in wrapped diff lines now allows selection of individual lines - #1551",
+      "[Improved] Add option to fetch when a user needs to pull changes from the remote before pushing - #2738 #5451",
+      "[Improved] Enable Git protocol v2 for fetch/push/pull operations - #6142",
+      "[Improved] Moving the mouse pointer outside of the visible diff while selecting a range of lines in a partial commit will now automatically scroll the diff - #658",
```

Tightened up, changed from future tense to present tense:

```suggestion
      "[Improved] Moving mouse pointer outside visible diff while selecting a range of lines in a partial commit now automatically scrolls the diff - #658",
```